src/modules/launcher/Plugins/Microsoft.Plugin.WindowWalker/Properties/Resources.resx

```diff
@@ -162,7 +162,7 @@
     <value>Process id</value>
   </data>
   <data name="wox_plugin_windowwalker_ExplorerInfoSubTitle" xml:space="preserve">
-    <value>Folder windows doesn't run in separate processes. (Klick to open Explorer properties.)</value>
+    <value>Folder windows doesn't run in separate processes. (Click to open Explorer properties.)</value>
```

Comment:
```suggestion
    <value>Folder windows do not run in separate processes. (Click to open Explorer properties.)</value>
```
should it be either single or plural?

Reply:
Ahh yeah. 'Folder windows' is plural. Do should come.